ci/build_docs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+ENV_YAML_DIR="$(mktemp -d)"
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key docs | tee "${ENV_YAML_DIR}/env.yaml"
+
+rapids-mamba-retry env create --yes -n docs
+conda activate docs
+
+rapids-print-env
+
+rapids-logger "Downloading artifacts from previous jobs"
+PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
```

Comment:
I don't think this command will work as expected on this PR.

The previous jobs in the workflow don't upload artifacts to S3, they use GitHub's native artifact storage.